OWML.ModHelper.Menus/ModInputCombinationMenu.cs

```diff
@@ -0,0 +1,205 @@
+                        }
+                    }
+                }
+                return result;
+            }
+            set
+            {
+                _selectables = new List<Selectable>();
+                CombinationElements.ForEach(element => element.Destroy());
+                CombinationElements.Clear();
+                foreach (var combination in value.Split('/'))
+                {
+                    AddCombinationElement(combination);
+                }
+                SelectFirst();
+                UpdateNavigation(_selectables);
+            }
+        }
+
+        private IModInputCombinationElement _combinationElementTemplate;
+
+        public ModInputCombinationMenu(IModConsole console) : base(console)
+        {
+            CombinationElements = new List<IModInputCombinationElement>();
+        }
+
+        public override void Open()
+        {
+            base.Open();
+        }
+
+        public void Initialize(Menu menu, IModInputCombinationElement combinationElementTemplate)
+        {
+            _combinationElementTemplate = combinationElementTemplate;
+
+            var blocker = menu.GetComponentsInChildren<GraphicRaycaster>().Single(x => x.name == "RebindingModeBlocker");
+            blocker.gameObject.SetActive(false);
+
+            var labelPanel = menu.GetValue<GameObject>("_selectableItemsRoot").GetComponentInChildren<HorizontalLayoutGroup>();
+            labelPanel.gameObject.SetActive(false);
+
+            var layoutGroup = menu.GetComponentsInChildren<VerticalLayoutGroup>().Single(x => x.name == "Content");
+            Initialize(menu, layoutGroup);
+
+            var saveButton = GetButton("UIElement-SaveAndExit");
+            var addButton = GetButton("UIElement-ResetToDefaultsButton");
+            var cancelButton = GetButton("UIElement-DiscardChangesButton");
+
+            saveButton.OnClick += OnSave;
+            addButton.OnClick += OnAdd;
+            cancelButton.OnClick += OnExit;
+
+            saveButton.SetControllerCommand(InputLibrary.confirm);
+            cancelButton.SetControllerCommand(InputLibrary.cancel);
+            addButton.SetControllerCommand(InputLibrary.setDefaults);
+
+            var localText = addButton.Button.gameObject.GetComponentInChildren<LocalizedText>(true);
+            if (localText != null)
+            {
+                GameObject.Destroy(localText);
+            }
+            var buttonWithHotkey = addButton.Button.gameObject.GetComponentInChildren<ButtonWithHotkeyImageElement>(true);
+            if (buttonWithHotkey != null)
+            {
+                buttonWithHotkey.SetPrompt(new ScreenPrompt(InputLibrary.setDefaults, "Add Alternative"));
+            }
+
+            Title = "Edit Combination";
+
+            GetButton("UIElement-CancelOutOfRebinding").Hide();
+            GetButton("UIElement-KeyRebinder").Hide();
+
+            for (int i = 0; i < layoutGroup.transform.childCount; i++)
```

Comment:
foreach(Transform child in layoutGroup.transform)

Reply:
K